### DISCUSSION TITLE
Force close lib's

### DISCUSSION THREAD

**Discussion by UNKNOWNNR** [thatskymod/Sky-CotL-Scripts#120] [General]
the libsmite is force closing while it started. why it doesn't work??

**Comment by thatskymod** [comment]
I have reviewed libsmite and removed it to avoid further confusion. It will be added back once a working update becomes available.

For now, there's [libLangit041.so](https://github.com/thatskymod/Sky-CotL-Scripts/blob/a45cca3705b53b9dbe481313a253c7ad492b5ac7/Canvas/Libs/libLangit041.so) for  Sky Live 0.26.5 (280889).

Your feedback helps keep the repository organized and user-friendly. Thank you for your contribution.
